tests/pyright-docker/Dockerfile

```diff
@@ -1,3 +1,3 @@
-FROM node:15
+FROM node:15.4.0-alpine3.10
```

Comment:
I'd suggest 14 because in the Node world, the even-numbered versions become stable and supported versions.